exporters/memory/CMakeLists.txt

```diff
@@ -21,19 +21,56 @@ install(
   FILES_MATCHING
   PATTERN "*.h")
 
+find_package(Protobuf)
+if(NOT Protobuf_FOUND)
+  message(WARNING "Skipping in memory metric exporter: protobuf not found")
+else()
+  if(NOT WITH_OTLP)
+    include(${PROJECT_SOURCE_DIR}/cmake/opentelemetry-proto.cmake)
+  endif()
+
+  add_library(
+    opentelemetry_exporter_in_memory_metric
+    src/in_memory_metric_exporter.cc ../otlp/src/otlp_metric_utils.cc
+    ../otlp/src/otlp_populate_attribute_utils.cc)
+
+  target_include_directories(
+    opentelemetry_exporter_in_memory_metric
+    PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>"
+           "$<INSTALL_INTERFACE:include>"
+           "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/../otlp/include>")
```

Comment:
`target_link_libraries(opentelemetry_exporter_in_memory_metric PUBLIC opentelemetry_otlp_recordable)` would be a better alternative here.But when we building without `WITH_OTLP`, `opentelemetry_otlp_recordable` will be unavailable.
Maybe this module should also be disabled when we build without `WITH_OTLP` now?